api-design-guidelines/index.md

```diff
@@ -93,7 +93,9 @@ is printed.
       great summary.
 
     * **Use a single sentence fragment** if possible, ending with a
-      period.  Do not use a complete sentence.
+      period.  Do not use a complete sentence.  A summary is like 
+      a dictionary definition; it is shown in the context of the 
+      API it describes.
```

Comment:
> Isn't this obvious?  I mean, the summary is in the doc comment, which is
right there next to the declaration.

It should be, and yet sometimes when I'm writing a doc comment I have to remind myself of this. (I think this mainly happens when I'm documenting a type—I have an impulse to start "The ImageSource struct…")

Maybe it can just be worded as "Don't restate the API's name—it'll be obvious from the context."